crates/ev-cli/src/main.rs

```diff
@@ -41,6 +41,7 @@ where
         serde_json::json!({
             "message": output.to_string(),
             "code": output.code(),
+            "is_error": is_error
```

Comment:
Should this not be implicit from the stream that it's writing to?

Reply:
That's true, thought it would be handy to be able to check in a CI pipeline but maybe not